src/gc/gc.cpp

```diff
@@ -15535,6 +15542,27 @@ void gc_heap::gc1()
 
     descr_generations (FALSE);
 
+#ifdef GCTRIMCOMMIT
+    if (g_pConfig->GetGCTrimCommit())
```

Comment:
Please query this config option by adding it to the `GCConfig` class in `gc\gcconfig.h`. and `gc\gcconfig.cpp`. Referencing `g_pConfig` directly from the GC keeps the runtime and the GC closely coupled, which we're trying to avoid.